Cargo.lock

```diff
@@ -27,7 +27,7 @@ version = "0.19.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
```

Comment:
This Cargo.lock seems to be pulling a lot of updates, did you run `cargo update` for everything ?

Reply:
I didn't, I will before I sign the next commit.